src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -180,28 +213,66 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
         STG_PanelOpps_CTRL.verifyOpportunityStageForPaymentAutoCloseIsActiveClosedWon(this);
         verifyScheduledJobs();
         STG_PanelERR_CTRL.verifyErrorSettings(this);
-        verifyOppContactRoles();
         verifyOppContactRolesSettings();
         verifyTriggerHandlers();
-        verifyNoMissingOppPayments();
         verifyLevels();
 
+        if (STG_Panel.stgService.stgErr.Disable_Record_Data_Health_Checks__c == false) {
```

Comment:
In the case the value is null, data record health check should not execute. Thus, let's check for `== true`.
Let's rename `generateDisableRecordHealthCheckMessages()` to `addDisableRecordHealthCheckWarnings()` or `addDisableRecordHealthCheckMessages()`.
```
        if (STG_Panel.stgService.stgErr.DisableRecordDataHealthChecks__c == true) {
            addDisableRecordHealthCheckWarnings();
        } else {
            executeDataRecordsHealthChecks();
        }
```